### PR TITLE
Spawn Widget Actor fix.

### PR DESCRIPTION
**Pull request by Tod-Rive**
Fixed "spawn widget actor" from the rive texture context menu.

### CODE REVIEW

**Review by schetle — COMMENTED** [review]
I would propose not to fix it in this way, but instead adjust the line 224.

It currently contains this:
`UTexture2DDynamic::StaticClass()`

Which will add this Spawn context menu option to all UTexture2DDynamic assets. This will cause errors on those kinds of assets, because RiveTextureObject will be invalid (so will RiveFile).

The fix here should probably be adjusting this line to be:
URiveFile::StaticClass()

Which sill set this context menu option on RiveFile assets.

Or we can adjust it to be specifically URiveTextureObject::StaticClass() if instead we require you to have a texture object and right-click that, instead of RiveFile.

Personally, I think it makes a bit more since to have it on the URiveFile; which would make the rest of these code changes unnecessary.

**Comment by Tod-Rive** [comment]
@schetle  That makes more sense. Moved the menu item back to RiveFile.

File: Source/RiveEditor/Private/Factories/RiveWidgetFactory.cpp [hunk 1]
```diff
@@ -199,8 +199,8 @@ namespace
 	{
 		if (const UContentBrowserAssetContextMenuContext* Context = UContentBrowserAssetContextMenuContext::FindContextWithAssets(MenuContext))
 		{
-			TArray<URiveFile*> RiveFiles = Context->LoadSelectedObjects<URiveFile>();
-			for (URiveFile* RiveFile : RiveFiles)
+			TArray<URiveFile*> URiveFiles = Context->LoadSelectedObjects<URiveFile>();
```
Comment (schetle) [inline Source/RiveEditor/Private/Factories/RiveWidgetFactory.cpp:202]: Just as a stylistic comment - Typically the class prefix (U, F, E, etc. that is required for Unreal-based types) aren't used in Variable Names themselves.

**Review by schetle — APPROVED** [review]

File: Source/RiveEditor/Private/Factories/RiveWidgetFactory.cpp (hunk 1 above)
Comment (Tod-Rive) [reply Source/RiveEditor/Private/Factories/RiveWidgetFactory.cpp:202]: Copy/Paste error. Good catch.

**Pull request merged**